pkg/controller/csr_check_test.go

```diff
@@ -422,6 +170,42 @@ func generateCertKeyPair(duration time.Duration, parentCertPEM, parentKeyPEM []b
 	return certOut.Bytes(), keyOut.Bytes(), nil
 }
 
+func createCR(commonName string, organizations []string, ipAddressess []net.IP, dnsNames []string) string {
+	defaultOrganizations := []string{"system:nodes"}
+	defaultIPAdresses := []net.IP{net.ParseIP("127.0.0.1"), net.ParseIP("10.0.0.1")}
+	defaultDnsNames := []string{"node1", "node1.local"}
```

Comment:
Would it be more obvious what the function is doing if you were required to pass these in rather than defaulting in here? When I see the chain of nils I'm left wondering what should be in there
eg 
```
goodCSR = createCR("system:node:test", defaultOrgs, defaultIPs, deafultDNSNames)
```